check_test.go

```diff
@@ -1,609 +1,694 @@
 package opslevel_test
 
 import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"strings"
 	"testing"
+	"text/template"
 
+	"github.com/Masterminds/sprig/v3"
 	ol "github.com/opslevel/opslevel-go/v2024"
 	"github.com/rocktavious/autopilot/v2023"
 )
 
-var checkCreateInput = ol.CheckCreateInput{
-	Name:     "Hello World",
-	Enabled:  ol.RefOf(true),
-	Category: ol.ID("Z2lkOi8vb3BzbGV2ZWwvQ2F0ZWdvcnkvNjA1"),
-	Level:    ol.ID("Z2lkOi8vb3BzbGV2ZWwvTGV2ZWwvMzE3"),
-	Notes:    ol.RefOf("Hello World Check"),
+// Temporary solution until repo wide testing is standardized
+type TmpCheckTestCase struct {
+	fixture autopilot.TestRequest
+	body    func(c *ol.Client) (*ol.Check, error)
 }
 
-var checkUpdateNotes = "Hello World Check"
+// Helper Data
+var (
+	templater = template.New("").Funcs(sprig.TxtFuncMap()).Delims("[%", "%]")
+
+	id = ol.ID("Z2lkOi8vb3BzbGV2ZWwvMTIzNDU2")
```

Comment:
Just an FYI there are ids for tests in [clientGQL_test.go](https://github.com/OpsLevel/opslevel-go/blob/main/clientGQL_test.go#L25)